src/SDKs/CustomerInsights/Management.CustomerInsights/Microsoft.Azure.Management.CustomerInsights.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <Description>Provides developers with libraries to create and manage Namespaces and manage Authorization Rules. Note: This client library is for CustomerInsights under Azure Resource Manager.</Description>
-    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <VersionPrefix>0.1.1-preview</VersionPrefix>
```

Comment:
@tjlvtao please remove all the other nuget package properties.
Only PackageId, AssemblyName, Description, PackageTags and versionPrefix stays from the first property group.
All the commented code should also be removed. This was kept for referencing old references, nuget references.